acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -7,7 +7,7 @@
 test_name "Install Packages"
 
 step "Install repositories on target machines..." do
-  sha = ENV['SHA']
+  sha = ENV['VERSION'] || ENV['SHA']
```

Comment:
hmmm. i hadn't noticed that.  VERSION is used in certain cases and seems more uniform than SHA which is not always what the value 'is'.  
in many jobs these days i think this is also called GIT_SHA.
argh...  attempts to move to uniformity are always punished